css/styles.css

```diff
@@ -0,0 +1,165 @@
+* {
+  font-family: Merriweather, Source Sans Pro, Oswald;
+  font-weight: lighter;
+}
+
+header {
+  background-color: black;
+  float: left;
+  width: 100%;
+  border-bottom: 3px solid yellow;
+  color: white;
+}
+
+nav {
+  text-align: right;
+  padding: 10px;
+}
+
+nav a {
+  color: white;
+  list-style: none;
+  padding-right: 25px;
+  text-decoration: none;
+}
+
+article {
+  background-color: red;
+  clear: both;
+}
+
+aside {
+  clear: both;
+  height: 45vh;
+  width: 100%;
+}
+
+footer {
+  background-color: black;
+  clear: both;
+  width: 100%;
+  height: 20px;
+  font-family: sans-serif;
+  color: white;
+  padding: 15px;
+}
+
+h1 {
+  font-family: sans-serif;
+  font-weight: bolder;
+  font-size: 36px;
+}
+
+.logo {
+  color: yellow;
+  font-size: 24px;
+  padding: 15px;
+}
+
+.impact-font {
+  font-family: Impact, Charcoal, sans-serif;
+  font-size: 18px;
+  text-transform: uppercase;
+}
+
+.calibri-font {
+  font-family: sans-serif;
+}
+
+.tagline {
+  background-color: lightgrey;
+  width: 100%;
+  height: 5%;
+  float: left;
+}
+
+.tagline p {
+  text-align: center;
+  vertical-align: middle;
+  font-style: italic;
+  font-size: 18px;
+}
+
+#taste {
+  color: grey;
+  text-decoration: line-through;
+}
+
+#are {
+  border-bottom: solid 2px yellow;
+}
+
+.hero {
+  background-image: url("../images/coworking.jpg");
+  background-size: 100%;
```

Comment:
@kvelez1 to remove the white space aound the image, try `background-size: cover` 🔨